pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -725,18 +735,19 @@ type RestoreCondition struct {
 // +k8s:openapi-gen=true
 // RestoreSpec contains the specification for a restore of a tidb cluster backup.
 type RestoreSpec struct {
-	// Cluster represents the tidb cluster to be restored.
-	Cluster string `json:"cluster"`
-	// Backup represents the backup object to be restored.
-	Backup string `json:"backup"`
-	// Namespace is the namespace of the backup.
-	BackupNamespace string `json:"backupNamespace"`
-	// SecretName is the name of the secret which stores
-	// tidb cluster's username and password.
-	TidbSecretName string `json:"tidbSecretName"`
-	// StorageClassName is the storage class for restore job's PV.
+	// To is the tidb cluster that needs to restore.
+	To TiDBAccessConfig `json:"to"`
+	// BackupPath is the location of the backup.
+	BackupPath string `json:"backupPath"`
+	// Type is the backup type for tidb cluster.
+	Type BackupType `json:"backupType,omitempty"`
```

Comment:
I think we can allow users to omit this field if there is only one storage provider to improve UX.

Reply:
The `BackupType` field is unused currently. Do you mean `StorageType`?